README.md

```diff
@@ -161,6 +161,84 @@ No changes are required to your actions, react-chrome-redux automatically adds t
 
 `react-chrome-redux` supports `onMessageExternal` which is fired when a message is sent from another extension, app, or website. By default, if `externally_connectable` is not declared in your extension's manifest, all extensions or apps will be able to send messages to your extension, but no websites will be able to. You can follow [this](https://developer.chrome.com/extensions/manifest/externally_connectable) to address your needs appropriately.
 
+## Custom Serialization
+
+You may wish to implement custom serialization and deserialization logic for communication between the background store and your proxy store(s). Chrome's message passing (which is used to implement this library) automatically serializes messages when they are sent and deserializes them when they are received. In the case that you have non-JSON-ifiable information in your Redux state, like a circular reference or a `Date` object, you will lose information between the background store and the proxy store(s). To manage this, both `wrapStore` and `Store` accept `serializer` and `deserializer` options. These should be functions that take a single parameter, the payload of a message, and return a serialized and deserialized form, respectively. The `serializer` function will be called every time a message is sent, and the `deserializer` function will be called every time a message is received. Note that, in addition to state updates, action creators being passed from your content script(s) to your background page will be serialized and deserialized as well.
+
+### Example
+For example, consider the following `state` in your background page:
+
+```js
+{todos: [
+    {
+      id: 1,
+      text: 'Write a Chrome extension',
+      created: new Date(2018, 0, 1)
+    }
+]}
+```
+
+With no custom serialization, the `state` in your proxy store will look like this:
+
+```js
+{todos: [
+    {
+      id: 1,
+      text: 'Write a Chrome extension',
+      created: {}
```

Comment:
Is this true? I swore it turned it into a string last time I checked... am I crazy?

Reply:
Vanilla `JSON.parse(JSON.stringify(date))` will give you a string, but Chrome's default serialization/deserialization for messaging gives you an empty object for some reason or another.

Reply:
Got it - must have been confusing the two. Thanks

Reply:
Wow this is really good to know. I could see us having a `serializers` dir with defaults for things like dates (not necessary for this PR ofc).